autoload/gh/github/repos.vim

```diff
@@ -56,3 +56,8 @@ function! gh#github#repos#get_repo(owner, repo) abort
   let url = printf('https://api.github.com/repos/%s/%s', a:owner, a:repo)
   return gh#http#get(url)
 endfunction
+
+function! gh#github#repos#get_assignees(owner, repo) abort
+  let url = printf('https://api.github.com/repos/%s/%s/assignees', a:owner, a:repo)
```

Comment:
I'll replace REST to GraphQL in the future, so I think we should use GraphQL API like gists implementation.

Reply:
@skanehira 
switched to use GraphQL from REST by [this commit](https://github.com/skanehira/gh.vim/pull/119/commits/026bf771249b04825b0ae1ff4f5924d8061abc0c).